src/page-building/career/career.stories.mdx

```diff
@@ -0,0 +1,51 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs';
+import { ApplicationProcess } from './application-process'; 
+import { Perks } from './perks'; 
+import { Openings } from './openings';
+import { Culture } from './culture'; import { CareerPage } from './career-page';
+
+<Meta title="Pages/Career" component={{ Image }} />
+
+# Career
```

Comment:
I like how I can quickly check the sections available on the career page. I used this while developing.